### ISSUE TITLE
Symmetry in _apply_recursion_rule_2

### ISSUE THREAD

**Issue by jharmse** [akelleh/causality#78]
The function loops over edges `(a,b)` and checks whether it should add an arrow from `a` to `b`, but it does not check whether it should add an arrow from `b` to `a`. Note that `self._g.edges()` has only one entry for each unordered pair.

I tried to submit a pull request, but I apparently I can't `push` a new branch to the repository.

**Comment by akelleh** [comment]
good catch! fixed this on master -- do you have a test case to run against it?

**Comment by jharmse** [comment]
Thank you for fixing it. I happened to notice the issue while looking at
the code, and I didn't develop a test case to show the difference.

Regards,
Jorgen Harmse.
